src/outpack/filestore.py

```diff
@@ -22,7 +23,7 @@ def get(self, hash, dst, *, overwrite=False):
         src = self.filename(hash)
         if not os.path.exists(src):
             msg = f"Hash '{hash}' not found in store"
-            raise Exception(msg)
+            raise FileNotFoundError(ENOENT, msg)
```

Comment:
Is just for convention, or is something else responding more nicely here?

Reply:
I use that to catch the error in `copy_files` to figure out when something is not in the store.